### ISSUE TITLE
No transition for new maximised windows

### ISSUE THREAD

**Issue by somepaulo** [ewlsh/dynamic-panel-transparency#105]
When opening new maximised windows (for apps that remember windows size, like Chrome or Firefox) the panel stays transparent.
Going to overview and back makes the panel opaque as it should be.

There's an error in `journalctl`:
```
JS ERROR: TypeError: St.get_slow_down_factor is not a function
fade_in@/home/somepaulo/.local/share/gnome-shell/extensions/dynamic-panel-transparency@rockon999.github.io/transitions.js:99:21
_check@/home/somepaulo/.local/share/gnome-shell/extensions/dynamic-panel-transparency@rockon999.github.io/intellifade.js:260:13
asyncCheck@/home/somepaulo/.local/share/gnome-shell/extensions/dynamic-panel-transparency@rockon999.github.io/intellifade.js:82:9
_windowRestacked@/home/somepaulo/.local/share/gnome-shell/extensions/dynamic-panel-transparency@rockon999.github.io/events.js:302:13
```

**Comment by rockon999** [comment]
Hi! This seems to be actually related to #104. Essentially, the corner support is broken in 3.34 - I'm working to fix it!

**Comment by cybereality** [comment]
I'm having the same problems since upgrading to Ubuntu 19.10.

**Comment by ewlsh** [comment]
Fixed in #126 for 3.36/3.38. Not planning to back port for 3.34 at this time.
